calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| ------------------------ | ------------------------------- | -------------------------------------------------------------------------------------- |
+| `input_lidar_pointcloud` | `sensor_msgs::msg::PointCloud2` | Lidar pointcloud for calibration. `input_lidar_pointcloud` is defined in the launcher. |
+| `input_radar_msg`        | `radar_msgs::msg::RadarTracks`  | Radar message for calibration, `input_radar_msg` is defined in the launcher.           |
+
+### Output
+
+| Name                          | Type                                   | Description                                               |
+| ----------------------------- | -------------------------------------- | --------------------------------------------------------- |
+| `lidar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the background pointcloud from lidar.           |
+| `lidar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the foreground pointcloud from lidar.           |
+| `lidar_colored_clusters`      | `sensor_msgs::msg::PointCloud2`        | Publishes colored pointcloud clusters from lidar.         |
+| `lidar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Publishes lidar detections.                               |
+| `radar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the background pointcloud from radar.           |
+| `radar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the foreground pointcloud from radar.           |
+| `radar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Publishes radar detections.                               |
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Publishes matched lidar and radar detection.              |
+| `tracking_markers`            | `visualization_msgs::msg::MarkerArray` | Publishes tracks of reflectors.                           |
+| `text_markers`                | `visualization_msgs::msg::Marker`      | Publishes text markers that show the calibration metrics. |
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Publishes calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Strat to extract the background model from radar and lidar data.                         |
```

Comment:
"Strat to"?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/3daad85342e764704a624b38d9b123fbd14c0645